### DISCUSSION TITLE
Disabling AutoImport for Layers Only

### DISCUSSION THREAD

**Discussion by Plinpod** [nuxt/nuxt#29988] [Questions]
Hello,

I have a nested folder structure for layers for organization purposes. Something along the lines of 

```
/layers/group1/layer1
/layers/group1/layer2

/layers/group2/layer1
/layers/group2/layer2
```

Everything works but AutoImport tries to import the group **group1** and **group2** folders, which I don't need. My nested layers are extended directly within the config. 

Is it possible to disable AutoImport for layers specifically or is there another folder structure that I should be using?

Thanks!

**Comment by angelhdzmultimedia** [comment]
You can create a base.layer.ts https://nuxt.com/docs/getting-started/layers#usage that sets auto imports false, and all your layers extend from that layer.
That way your main app has auto imports, but not your layers. I haven't tried this because I disable auto imports altogether, i prefer explicit imports. I'm also lately inclined to use namespaces, so I import the vue or nuxt or h3 namespaces, and use everything from there.

Wish you success! 👋💚
